<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ThumbnailContainer</class>
 <widget class="QWidget" name="ThumbnailContainer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>227</width>
    <height>705</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>227</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#ThumbnailContainer{
background-color:#F9F9F9;
border:0px solid #F9F9F9;
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="addBtn">
        <property name="maximumSize">
         <size>
          <width>12</width>
          <height>24</height>
         </size>
        </property>
        <property name="text">
         <string>+</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushBtn">
        <property name="maximumSize">
         <size>
          <width>12</width>
          <height>66</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton#pushBtn{
  border-image:url(:/res/images/fold.png);
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="delBtn">
        <property name="maximumSize">
         <size>
          <width>12</width>
          <height>24</height>
         </size>
        </property>
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="listWidget">
     <property name="minimumSize">
      <size>
       <width>215</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QListWidget#listWidget{
 background-color:#F9F9F9;
 border:0px solid #F9F9F9;
 outline:0px;
}

QListWidget#listWidget::Item{
 background-color:#F9F9F9;
}

QListWidget#listWidget::Item:hover { 
 background-color:#F9F9F9;
}

QListWidget#listWidget::Item:selected { 
 background-color:#F9F9F9;
}

QScrollBar::handle:vertical
{
    width:8px;
    background:rgb(232, 232, 232);
    border-radius:4px;  
    min-height:20;
}</string>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAsNeeded</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustIgnored</enum>
     </property>
     <property name="autoScrollMargin">
      <number>0</number>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::NoSelection</enum>
     </property>
     <property name="verticalScrollMode">
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="demo.qrc"/>
 </resources>
 <connections/>
</ui>
